plasmapy/formulary/ionization.py

```diff
@@ -1,8 +1,8 @@
 """ This module gathers functions relating to ionization states and the properties thereof.
 """
-__all__ = ["ionization_balance"]
+__all__ = ["ionization_balance", "Z_bal_", "Saha"]
```

Comment:
alphabetize...this does defined the order in which these function appear in the documentation table